api/server/server.go

```diff
@@ -19,44 +20,54 @@ type Server struct {
 
 	node       *handlers.Node
 	validators *handlers.Validators
+	tokenAuth  *jwtauth.JWTAuth
 }
 
 func New(
 	logger *zap.Logger,
 	addr string,
 	node *handlers.Node,
 	validators *handlers.Validators,
+	tokenAuth *jwtauth.JWTAuth,
 ) *Server {
 	return &Server{
 		logger:     logger,
 		addr:       addr,
 		node:       node,
 		validators: validators,
+		tokenAuth:  tokenAuth,
 	}
 }
 
 func (s *Server) Run() error {
+	s.logger.Info("Serving SSV API", zap.String("addr", s.addr))
+	server := &http.Server{
+		Addr:         s.addr,
+		Handler:      s.SetRoutes(),
+		ReadTimeout:  12 * time.Second,
+		WriteTimeout: 12 * time.Second,
+	}
+	return server.ListenAndServe()
+}
+
+func (s *Server) SetRoutes() chi.Router {
```

Comment:
This should be a private method since it does not need to be called from outside.

I think we can just call it `routes()` since it also returns the routes rather than just 'set' them.